jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -197,9 +234,9 @@ pipeline {
                 } else {
                     // upload log only in case of build failure
                     def guardWords = ["gitlab.*?\\.com", "urm.*?\\.com"]
-                    def logPattern = "### BEGIN OF TEST LOG ###.*### END OF TEST LOG ###"
+                    guardWords.add("nvidia-smi(?s)(.*?)(?=jenkins/version-def.sh)") // hide GPU info
```

Comment:
looks like this will filter out the UT log part w/ new commit?

Reply:
nice catch! Will fix it